sherpa/plot/__init__.py

```diff
@@ -2731,6 +2875,20 @@ def _region_init(self, fit, par0, par1):
                 raise ConfidenceErr(
                     'badarg', 'Parameter limits', 'a list') from None
 
+        # We ignore the return value for min/max
+        check2(self.min, "Parameter limits")
+        check2(self.max, "Parameter limits")
+        nloop = check2(self.nloop, "Nloop parameter", opt=False)
+        delv = check2(self.delv, "delv parameter")
+
+        if numpy.any(nloop <= 1):
+            raise ConfidenceErr('badarg', 'Nloop parameter',
+                                'a list with elements > 1')
+
+        if delv is not None and numpy.any(delv <= 0):
+            raise ConfidenceErr('badarg', 'delv parameter',
+                                'a list with elements > 0')
```

Comment:
So, the check here doesn't enforce the "list of size 2" check (this check has already been done by the time this is called), so I'm a bit reticent about changing the error message (since some other later refactoring may change things so the check is done before the size check, for instance).